pkg/pb/health.proto

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+option go_package = "pb";
+
+package health;
+
+import "gogo.proto";
+import "pointerdb.proto";
+
+service Health {
```

Comment:
Not sure whether `Health` is the right name for the service. There's also `Inspector`, but I'm not sure whether that's any better.

Reply:
done